stubs/redis/redis/cluster.pyi

```diff
@@ -106,47 +106,49 @@ class RedisCluster(AbstractRedisCluster, RedisClusterCommands[_StrType], Generic
     def close(self) -> None: ...
 
 class ClusterNode:
-    host: Any
-    port: Any
-    name: Any
-    server_type: Any
-    redis_connection: Any
-    def __init__(self, host, port, server_type: Any | None = ..., redis_connection: Any | None = ...) -> None: ...
-    def __eq__(self, obj): ...
+    host: str
+    port: int
+    name: str
+    server_type: str | None
+    redis_connection: Redis[Any] | None
```

Comment:
```suggestion
    redis_connection: Redis[Incomplete] | None
```

Reply:
It is `Any`, because it can be `str` or `bytes`, but we don't care about it here.

Reply:
Theoretically, couldn't somebody at some point make `ClusterNode` generic, such that whether it is `Redis[str]` or `Redis[bytes]` could be inferred from the type passed into the `redis_connection` parameter of `ClusterNode.__init__`?

Reply:
I am not sure about it 🤔 
I don't know how redis cluster behaves in terms of encoding.

Reply:
If we don't know, then I'd prefer for us to go with `Incomplete` rather than `Any` ;) I think we should use `Any` only when we know for sure that there's no other viable alternative :)